cmd/gcp-controller-manager/ca_cache_test.go

```diff
@@ -41,6 +41,7 @@ func TestCACacheVerify(t *testing.T) {
 			}
 		})
 		for desc, invalidCert := range ca.invalidCerts {
+			invalidCert := invalidCert
```

Comment:
Should this be in this pull request? Or is this from something else you are working on.